billing/scout-billing.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Docker Scout billing
+description: Learn how to buy Docker Scout Team in Docker Hub and manage your subscription
+keywords: payments, billing, subscription
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout lets users secure their software supply chain and continuously observe and improve their security posture. Docker Scout is free for up to 3 repositories. You can buy Docker Scout Team to turn on Docker Scout for additional repositories. In this section, learn how to buy Docker Scout Team in Docker Hub for your personal account or for an organization.
+
+## Personal account
+
+1. Select your username in the top-right corner of Docker Hub.
+2. From the drop-down menu select **Billing**.
+3. In the **Plan** section, navigate to the **Docker Scout** plan section. The default plan is **Docker Scout Free**. 
+4. Next to the plan name, select **Buy now**.
+5. From the **Purchase Scout Team** drop-down menu, select the name of your personal account.
+6. This redirects you to the payment processing page where you can enter your email, card information, and billing address.
+7. Update the quantity of repositories you want to use with Docker Scout. See [Docker Scout subscriptions](../subscription/scout-details.md) to learn more about the number of repositories available for each plan.
+8. Select annual or monthly billing.
+9. Select **Subscribe**.
+
+Once your purchase is complete, you receive a confirmation email and a copy of your invoice.
+
+## Organization
+
+1. In Docker Hub, select **Organizations**. 
+2. Select the organization namespace. 
+3. Select the **Billing** tab.
+4. In the **Plan** section, navigate to the **Docker Scout** plan section. The default plan is **Docker Scout Free**. 
+5. Next to the plan name, select **Buy now**.
+6. From the **Purchase Scout Team** drop-down menu, select the name of your organization.
```

Comment:
I'm not 100% sure if Business is preferred to Team based on multiple teams on an org since Scout just applies to repositories, @ob1dev or @kellyfreet-docker can you weigh in? Just want to make sure my understanding is correct!

Reply:
Scout subscription applies to an account, whether a user or an organization. If a customer has two or more organization accounts, s/he will have to pay for each when selecting Scout Team tier.

With Scout Business a customer could negotiate a deal with the sales team to pay once for multiple organization accounts.

To me, Allie's point is valid.